rocky/components/modal/script.js

```diff
@@ -0,0 +1,30 @@
+import { onDomReady } from "../js/imports/utils.js";
+
+onDomReady(initDialogs);
+
+export function initDialogs() {
+  let modal_components = document.querySelectorAll(".modal-wrapper");
+
+  modal_components.forEach((modal) => {
+    modal
+      .querySelector("button.modal-trigger")
```

Comment:
This could probably be just a >button query. unless we want users to be able to include open buttons inside the dialog itself. By selecting based on hierarchy, we can drop the modal-trigger class.

Reply:
I've come to the realisation that we might want to facilitate other triggers instead of assuming it will always be a button. So I was kinda thinking the opposite direction: removing the `button` part, but that would undermine your suggestion _a bit_.

Reply:
That's fine too, but make sure to do a querySelectorAll to allow for more than one 'activator'